### ISSUE TITLE
Compile fails on Angular 8 with enableIvy:true

### ISSUE THREAD

**Issue by abbas999** [bootsoon/ng-circle-progress#86]
Build a project that uses ng-circle-progress v1.4.0 on Angular 8 after adding
```
"angularCompilerOptions": {
  "enableIvy": true
}
```
to tsconfig.app.json. Compile fails with the following error:
> ERROR in ENOENT: no such file or directory, open '/my_angular_project/node_modules/ng-circle-progress/ng-circle-progress.js'

This is because package.json mentions
> "module": "ng-circle-progress.js"

while no such file exists. If I manually change ng-circle-progress.js to index.js, all works fine.

**Comment by DanielCardona** [comment]
The same here, thanks @abbas999 !

**Comment by abbas999** [comment]
I think this can be resolved by using ng-packgr (https://github.com/ng-packagr/ng-packagr) to push to npm.

**Comment by Ezard** [comment]
I believe that this has now been fixed by #76

**Comment by omahjoub** [comment]
Hi all,

Same problem here with ivy enabled and `ng-circle-progress: ~1.4.1`

 `ERROR in ENOENT: no such file or directory, open 'SOME_PATH/node_modules/ng-circle-progress/ng-circle-progress.js'`

```Angular CLI: 8.1.2
Node: 12.6.0
OS: darwin x64
Angular: 8.1.2
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... platform-server, router, service-worker

Package                                    Version
--------------------------------------------------------------------
@angular-devkit/architect                  0.801.2
@angular-devkit/build-angular              0.801.2
@angular-devkit/build-optimizer            0.801.2
@angular-devkit/build-webpack              0.801.2
@angular-devkit/core                       8.1.2
@angular-devkit/schematics                 8.1.2
@angular/pwa                               0.801.2
@ngtools/webpack                           8.1.2
@nguniversal/express-engine                8.1.1
@nguniversal/module-map-ngfactory-loader   8.1.1
@schematics/angular                        8.1.2
@schematics/update                         0.801.2
rxjs                                       6.4.0
typescript                                 3.4.5
webpack                                    4.35.2
```

@bootsoon Is there any fix planned for this? Thx.
